packages/grid/x-data-grid/src/hooks/features/rows/useGridParamsApi.ts

```diff
@@ -133,12 +133,6 @@ export function useGridParamsApi(apiRef: React.MutableRefObject<GridPrivateApiCo
     (id, field) => {
       const colDef = apiRef.current.getColumn(field);
 
-      if (process.env.NODE_ENV !== 'production') {
-        if (!colDef && !warnedOnceMissingColumn) {
-          warnMissingColumn(field);
-        }
-      }
-
```

Comment:
This makes our test suite fail due to the warning. Similar problem than the `try/catch` above, accessing an unexistent cell with `.getCellParams()` is considered an error.